pkg/kubectl/cmd/rollingupdate.go

```diff
@@ -172,10 +170,19 @@ func RunRollingUpdate(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, arg
 		if err != nil {
 			return err
 		}
+
```

Comment:
Updating examples good too, but I was referring specifically to this
```
		Use: "rolling-update OLD_CONTROLLER_NAME -f NEW_CONTROLLER_SPEC",
```
It should be updated to
```
		Use: "rolling-update OLD_RC_NAME ([NEW_RC_NAME] --image=IMAGE | -f NEW_RC_FILENAME)",
```